src/main/resources/templates/georchestra/fragments/pmlinks.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+
+    <title>Password Management Links Fragment</title>
+    <link href="../../static/css/cas.css" rel="stylesheet" th:remove="tag" />
+</head>
+
+<body>
+    <div th:fragment="pmlinks" id="pmlinks" class="my-2">
+        <form th:if="${accountRegistrationEnabled}" method="post" id="accountMgmtSignupForm">
+            <input type="hidden" name="execution" th:value="${flowExecutionKey}" />
+            <input type="hidden" name="_eventId" value="signup" />
+            <span class="mdi mdi-account-group"></span>
+            <a id="accountSignUpLink" th:utext="#{screen.acct.button.signUp}" href="javascript:void(0)"
+               onclick="document.getElementById('accountMgmtSignupForm').submit()" />
+            <br><br>
+        </form>
+        <form th:if="${passwordManagementEnabled}" method="post" id="passwordManagementForm">
+            <input type="hidden" name="execution" th:value="${flowExecutionKey}" />
+            <input type="hidden" name="_eventId" value="resetPassword" />
+            <span class="mdi mdi-lock-open"></span>
+            <a id="forgotPasswordLink" th:utext="#{screen.pm.button.resetPassword}" href="javascript:void(0)"
+                onclick="document.getElementById('passwordManagementForm').submit();" />
+            <br><br>
+        </form>
+        <form th:if="${passwordManagementEnabled}" method="post" id="passwordManagementUserForm">
+            <input type="hidden" name="execution" th:value="${flowExecutionKey}" />
+            <input type="hidden" name="_eventId" value="forgotUsername" />
+            <a id="forgotUsernameLink" th:utext="#{screen.pm.button.forgotUsername}" href="javascript:void(0)"
+                onclick="document.getElementById('passwordManagementUserForm').submit();" />
+            <br><br>
+        </form>
+        <div th:unless="${passwordManagementEnabled}" id="resetPasswordLink">
+            <a href="/console/account/new">S'inscrire</a> -
```

Comment:
@pmauduit had not seen this one. Do you think we could add one translation ?

Reply:
It should be possible, by overriding this file:
https://github.com/apereo/cas/blob/master/support/cas-server-support-thymeleaf/src/main/resources/messages_fr.properties

then using the following thymeleaf directive referencing a key:

```
<label for="publicWorkstation" th:utext="#{screen.welcome.label.publicstation}"/>
```

Reply:
maybe you don't even need to override the file, as there is already a key that could be used:
https://github.com/apereo/cas/blame/master/support/cas-server-support-thymeleaf/src/main/resources/messages_fr.properties#L15

Reply:
i *think* i had seen this translation key, but _"S'enregistrer"_ is a bit **'meh'**  compared to _'créer un compte'_ ou _"S'inscrire"_ ... oh well :)

Reply:
Ah, thanks @pmauduit I hadn't seen that key nor realized that we could overwrite `messages_fr.properties` within the theme, as well, but I'm testing, and it works. Now, the question is just if the default translation of "S'enregistrer" is good enough French (hard for me to judge) or if we prefer to overwrite the entire `messages_fr.properties`  translations (which seems necessary to me in that case) to be able to replace it by "S'inscrire".

Reply:
=> added a small PR for that #5